lighthouse-core/lib/log.js

```diff
@@ -129,7 +129,40 @@ class Log {
     Log.events.issueStatus(title);
     return Log._logToStdErr(`${title}:verbose`, Array.from(arguments).slice(1));
   }
+
+  /**
+   * Add surrounding escape sequences to turn a string green when logged.
+   * @param {string} str
+   * @return {string}
+   */
+  static greenify(str) {
+    return `${Log.green}${str}${Log.reset}`;
+  }
+
+  /**
+   * Add surrounding escape sequences to turn a string red when logged.
+   * @param {string} str
+   * @return {string}
+   */
+  static redify(str) {
+    return `${Log.red}${str}${Log.reset}`;
+  }
+
 }
+
+Log.green = '\x1B[32m';
```

Comment:
Any reason why you don't make this constants? They don't need to be on the class.